examples/Design-class-DADesign.R

```diff
@@ -1,10 +1,11 @@
 # nolint start
```

Comment:
please remove the `nolint` start and end because we now want this file to be checked by the linter